git/plan.sh

```diff
@@ -1,22 +1,32 @@
 pkg_name=git
 pkg_version=2.7.4
 pkg_origin=core
-pkg_license=('gplv2')
+pkg_description="Git is a free and open source distributed version control
+  system designed to handle everything from small to very large projects with
+  speed and efficiency."
```

Comment:
Are we happy with multiline descriptions? I've been using shorter lines. Here's a good example for this one `"Distributed revision control system"`
I bet people kind of know what they are asking for in this environment. 😁